src/library_async.js

```diff
@@ -219,6 +219,22 @@ mergeInto(LibraryManager.library, {
     );
   },
 
+  emscripten_fiber_create: function() {
+    throw 'emscripten_fiber_create is not implemented for fastcomp ASYNCIFY';
+  },
+  emscripten_fiber_create_from_current_context: function() {
+    throw 'emscripten_fiber_create_from_current_context is not implemented for fastcomp ASYNCIFY';
+  },
+  emscripten_fiber_recycle: function() {
+    throw 'emscripten_fiber_recycle is not implemented for fastcomp ASYNCIFY';
+  },
+  emscripten_fiber_free: function() {
```

Comment:
Since the create function is so named perhaps this should be called `destroy` for parity?    Or maybe `new` + `delete` as a not to C++, or maybe there is precendent elsewhere in the emscripten API?

Reply:
I called it `_create` mostly because that's what the old coroutine API had, and that one didn't have a free/destroy function. I'm fine with renaming `_free` into `_destroy`. There is one precedent for `create`/`destroy` in the emscripten C API (`emscripten_create_worker`/`emscripten_destroy_worker`), and there's some more in the JS runtime (e.g. `FS.createNode`/`FS.destroyNode`).

Reply:
Renamed into _destroy